src/ad.jl

```diff
@@ -0,0 +1,18 @@
+const ADSUPPORT = (:ForwardDiff, :Zygote)
```

Comment:
Please consider keeping this file under `contrib` folder, i.e. `contrib/ad.jl`. This is mostly to keep the ad and other misc stuff clearly separated from HMC code.  

The folder `contrib` is a bit abused at the moment, we might want to add a `utilities` folder in the future.

Reply:
OK, I changed it :+1: